.eslintrc

```diff
@@ -74,20 +76,17 @@
         }],
         "no-self-compare": "error",
         "no-undef-init": "error",
-        "no-unused-vars": "off", // Override eslint:recommended rule
-        "no-useless-catch": "error",
         "no-useless-computed-key": "error",
         "no-useless-return": "error",
-        "no-with": "error",
         "prefer-const": ["error", {
             "destructuring": "any",
             "ignoreReadBeforeAssign": true
-          }],
+        }],
         "template-curly-spacing": "error",
         "yoda": "error",
 
         // Want to enable if there is consent
-        "no-shadow": "off", // 20 violations https://eslint.org/docs/rules/no-shadow
+        "no-shadow": "off", // 20 violations https://eslint.org/docs/rules/no-shadow	
```

Comment:
Are we still interested into enbabling those rules? 
I am fine making those changes, in another PR.

Reply:
I think we are!